src/app/interview-service/interview.service.spec.ts

```diff
@@ -1,12 +1,20 @@
 import { TestBed } from '@angular/core/testing';
-
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 import { InterviewService } from './interview.service';
 
 describe('InterviewService', () => {
-  beforeEach(() => TestBed.configureTestingModule({}));
+let interviewService: InterviewService;
```

Comment:
This service should also have more tests in another PR.

Reply:
We are working on those tests on another branch. See above.